src/OpenTelemetry.Exporter.Prometheus.AspNetCore/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Add resource attributes as tags for Prometheus exporters with filter
+  ([#3087](https://github.com/open-telemetry/opentelemetry-dotnet/pull/5407))
```

Comment:
Hmm, the link points to a different PR and there is a mismatch 3087/5407.

Reply:
My bad, fixed now.